js/sum.js

```diff
@@ -0,0 +1,47 @@
+window.addEventListener('DOMContentLoaded', () => {
+    let addBtn = document.getElementById('add-btn');
+    let newNum = document.getElementById('new-num');
+    let numsContainer = document.getElementById('added-nums-container');
+    let sumElem = document.getElementById('sum');
+    let isOdd = false;
+
+    addBtn.addEventListener('click', () => {
+        if (newNum.value === '') {
+            alert('Enter a number, please!');
+            return;
+        }
+
+        isOdd = !isOdd;
+        let currentElem = document.createElement('div');
+        isOdd ? currentElem.setAttribute('class', 'current-added-elem odd') : currentElem.setAttribute('class', 'current-added-elem');
+        let addedNum = document.createElement('span');
+        let deleteBtn = document.createElement('button');
+        deleteBtn.style.all = 'unset';
+        deleteBtn.innerText = '(delete)';
+        deleteBtn.setAttribute('id', 'delete');
+        deleteBtn.setAttribute('class', 'delete-btn');
```

Comment:
lets try to be a bit more 'functional'. We can extract the code above in a separate function that will create a new dom element(without appending it), add some text and attributes and return this element.